### PR TITLE
Mention the comment (ignorable) column function in translation csv

### PR DESCRIPTION
**Pull request by jsjtxietian**
As we have the https://github.com/godotengine/godot-proposals/issues/8183 done, this needs to be documented as akien-mga suggests https://github.com/godotengine/godot/pull/84569#issuecomment-1953825638

### CODE REVIEW

File: tutorials/assets_pipeline/importing_translations.rst [hunk 1]
```diff
@@ -57,25 +57,26 @@ CSV files must be formatted as follows:
...
-| QUOTE | "Hello" said the man. | "Hola" dijo el hombre. | 「こんにちは」男は言いました |
-+-------+-----------------------+------------------------+------------------------------+
++-------+-----------------------+------------------------+------------------------------+------------------------------+
+| keys  | en                    | es                     | ja                           | _character                   |
++=======+=======================+========================+==============================+==============================+
+| GREET | Hello, friend!        | Hola, amigo!           | こんにちは                   | John                          |
++-------+-----------------------+------------------------+------------------------------+------------------------------+
+| ASK   | How are you?          | Cómo está?             | 元気ですか                   | Jade                          |
++-------+-----------------------+------------------------+------------------------------+------------------------------+
+| BYE   | Goodbye               | Adiós                  | さようなら                   | Nancy                         |
```
Comment (AThousandShips) [inline tutorials/assets_pipeline/importing_translations.rst:76]:
```suggestion
| GREET | Hello, friend!        | Hola, amigo!           | こんにちは                    | John                          |
+-------+-----------------------+------------------------+------------------------------+------------------------------+
| ASK   | How are you?          | Cómo está?             | 元気ですか                    | Jade                          |
+-------+-----------------------+------------------------+------------------------------+------------------------------+
| BYE   | Goodbye               | Adiós                  | さようなら                    | Nancy                         |
```

Comment (jsjtxietian) [reply tutorials/assets_pipeline/importing_translations.rst]: Thanks!

**Comment by jsjtxietian** [comment]
Interesting, the ci still failed because malformedt table.

File: tutorials/assets_pipeline/importing_translations.rst [hunk 2]
```diff
@@ -57,25 +57,26 @@ CSV files must be formatted as follows:
...
++=======+=======================+========================+==============================+==============================+
+| GREET | Hello, friend!        | Hola, amigo!           | こんにちは                    | John                          |
++-------+-----------------------+------------------------+------------------------------+------------------------------+
+| ASK   | How are you?          | Cómo está?             | 元気ですか                    | Jade                          |
++-------+-----------------------+------------------------+------------------------------+------------------------------+
+| BYE   | Goodbye               | Adiós                  | さようなら                    | Nancy                         |
++-------+-----------------------+------------------------+------------------------------+------------------------------+
+| QUOTE | "Hello" said the man. | "Hola" dijo el hombre. | 「こんにちは」男は言いました     | Tian                         |
++-------+-----------------------+------------------------+------------------------------+------------------------------+
```
Comment (AThousandShips) [inline tutorials/assets_pipeline/importing_translations.rst:79]:
```suggestion
+-------+-----------------------+------------------------+------------------------------+------------------------------+
| keys  | en                    | es                     | ja                           | _character                   |
+=======+=======================+========================+==============================+==============================+
| GREET | Hello, friend!        | Hola, amigo!           | こんにちは                    | John                         |
+-------+-----------------------+------------------------+------------------------------+------------------------------+
| ASK   | How are you?          | Cómo está?             | 元気ですか                    | Jade                         |
+-------+-----------------------+------------------------+------------------------------+------------------------------+
| BYE   | Goodbye               | Adiós                  | さようなら                    | Nancy                        |
+-------+-----------------------+------------------------+------------------------------+------------------------------+
| QUOTE | "Hello" said the man. | "Hola" dijo el hombre. | 「こんにちは」男は言いました    | Tian                        |
+-------+-----------------------+------------------------+------------------------------+------------------------------+
```
My bad missed some, it's very hard to see on most formats

Comment (jsjtxietian) [reply tutorials/assets_pipeline/importing_translations.rst:79]: It‘s definitely not your bad, many thanks for your generous help!

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks!

**Pull request merged**